_includes/themes/zeppelin/_navigation.html

```diff
@@ -1,81 +1,98 @@
-    <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
-      <div class="container">
-        <div class="navbar-header">
-          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a class="navbar-brand" href="/">
-            <img src="/assets/themes/zeppelin/img/zeppelin_logo.png" width="50" alt="I'm zeppelin">
-            Apache Zeppelin
-          </a>
-        </div>
-        <nav class="navbar-collapse collapse" role="navigation">
-          <ul class="nav navbar-nav navbar-right">
-            {% assign pages_list = site.pages %}
-            {% assign group = 'nav-right' %}
-            {% include JB/pages_list %}
-
-            <!-- Download -->
-            <li id="docs">
-              <a href="#" data-toggle="dropdown" class="dropdown-toggle">Download<b class="caret"></b></a>
-              <ul class="dropdown-menu">
-                <li><a href="/download.html">Download Zeppelin</a></li>
-                <li><a href="/supported_interpreters.html">Supported Interpreters</a></li>
-              </ul>
-            </li>
+<div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/">
+        <img src="/assets/themes/zeppelin/img/zeppelin_logo.png" style="margin-top: -6px;" width="50" alt="I'm zeppelin">
+          <span style="margin-left: 5px;"> Apache Zeppelin </span>
+      </a>
+    </div>
+    <nav class="navbar-collapse collapse" role="navigation">
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="/docs/0.8.0-SNAPSHOT/install/install.html">Quick Start</a></li>
```

Comment:
@Leemoonsoo Thanks for good point!
Let me update this and the following comment as well.